lib/react_client.dart

```diff
@@ -906,35 +906,6 @@ _setValueToProps(Map props, val) {
   }
 }
 
-/// Convert bound values to pure value and packed onChange function
-@Deprecated('6.0.0')
-_convertBoundValues(Map args) {
-  var boundValue = args['value'];
-
-  if (boundValue is List) {
-    _setValueToProps(args, boundValue[0]);
-    args['value'] = boundValue[0];
-    var onChange = args['onChange'];
-
-    // Put new function into onChange event handler.
-    // If there was something listening for that event, trigger it and return its return value.
-    args['onChange'] = (event) {
-      boundValue[1](_getValueFromDom(event.target));
-
-      if (onChange != null) {
-        return onChange(event);
-      }
-    };
-  }
-}
-
-void _convertRefValue(Map args) {
```

Comment:
Seems like this is still being called on line 865

Reply:
Reinstated in force-pushed merge